definitions/npm/styled-components_v2.x.x/flow_v0.53.x-/styled-components_v2.x.x.js

```diff
@@ -7,25 +7,26 @@ type $npm$styledComponents$TaggedTemplateLiteral<R> = {| (Array<string>, $npm$st
 
 // ---- FUNCTIONAL COMPONENT DEFINITIONS ----
 type $npm$styledComponents$ReactComponentFunctional<Props, DefaultProps = *> = {
-  (Props): React$Element<*>,
+  (Props): React$Node<*>,
   defaultProps: DefaultProps,
 }
 
-type $npm$styledComponents$ReactComponentFunctionalUndefinedDefaultProps<Props> = { (Props): React$Element<*> }
+type $npm$styledComponents$ReactComponentFunctionalUndefinedDefaultProps<Props> = { (Props): React$Node<*> }
```

Comment:
`React$Node` [doesn't take a type argument](https://github.com/facebook/flow/blob/v0.54.0/lib/react.js#L16).

Reply:
Man, you're on top of this! :)

Reply:
Wow, that was fast.